README.md

```diff
@@ -331,7 +331,7 @@ This is just a rough summary. For more details, see [DevHistory.md](doc/DevHisto
 
 - 2021-04-05 New stable release: 1.7.1
 
-  - Added a "Discover" toplevel menu with actions to easily find
+  - Added a "Discover" top-level menu with actions to easily find
```

Comment:
No.

There are all kinds of possible spellings for this, depending on the variant of English (US, UK, rest of the world). I chose mine.